resources/js/src/app/services/ValidationService.js

```diff
@@ -200,6 +204,42 @@ function _hasValue($formControl)
     return $.trim($formControl.val()).length > 0;
 }
 
+/**
+ * @param {object} $formControl - Input inside Formular
+ * @returns value is valid date
+ */
+function _isValidDate($formControl)
+{
+    var string = $formControl.val();
+    var match = string.match(/^(?:(\d{1,2})[.\/-](\d{1,2})[.\/-](\d{4}))|(?:(\d{4})[.\/-](\d{1,2})[.\/-](\d{1,2}))$/);
+
+    // If match is null date is not valid
+    if (isNull(match))
+    {
+        return false;
+    }
+
+    // Check if year,month,day is valid
+    var currentYear = new Date().getFullYear();
+
+    // YYYY-MM-DD Format
+    if (isUndefined(match[3]))
+    {
+        if ((match[4] >= 1901 && match[4] <= currentYear) && (match[5] >= 1 && match[5] <= 12) && (match[6] >= 1 && match[6] <= 31))
+        {
+            return true;
+        }
+    }
+
+    // DD-MM-YYYY Format
+    if ((match[1] >= 1 && match[1] <= 31) && (match[2] >= 1 && match[2] <= 12) && (match[3] >= 1901 && match[3] <= currentYear))
```

Comment:
Macht es an der Stelle sinn, dass man alles darauf prüft, nicht in der Zukunft zu liegen? So wie es aktuell ist würde ja nur das Jahr nicht in der Zukunft liegen können aber Tag/Monat schon.

Reply:
Es wird jetzt nur noch geprüft ob es ein validates Datum ist. Nicht ob es sich um ein valides Geburtsdatum handelt.